components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngine.kt

```diff
@@ -42,6 +44,49 @@ class SystemEngine(
      */
     override fun name(): String = "System"
 
+    /**
+     * See [Engine.errorTypeFromCode]
+     */
+    override fun errorTypeFromCode(errorCode: Int, errorCategory: Int?): ErrorPages.ErrorType =
+    // Chromium's mapping (internal error code, to Android WebView error code) is described at:
```

Comment:
Can we fix the indentation so that the comments fall in-line with the start of the `when` block?